applications/ConstitutiveLawsApplication/custom_constitutive/composites/rule_of_mixtures_law.cpp

```diff
@@ -402,8 +395,8 @@ double& ParallelRuleOfMixturesLaw<TDim>::GetValue(
 			}
 		}
 	} else if (rThisVariable == CYCLES_TO_FAILURE){
-        double max_stress_relative_error;
-        double min_value=1.0e20;
+        //double max_stress_relative_error;
+        double min_value = std::numeric_limits<double>::infinity();
```

Comment:
```suggestion
        double min_value = std::numeric_limits<double>::max();
```

Nothing wrong with infinity but for debugging reasons is much better to handle real numbers in case something unexpected happens